src/hid.rs

```diff
@@ -0,0 +1,200 @@
++// Copyright � 2015, Alex Daniel Jones
++// Licensed under the MIT License <LICENSE.md>
++//! Taken from hidclass.h, hidsdi.h and hidpi.h
```

Comment:
Ah I was wondering about that; Didn't want to clutter up the src folder with multiple files. Will correct.